userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1384,6 +1384,11 @@ int32_t scap_bpf_close(struct scap_engine_handle engine)
 		handle->program_fd = -1;
 	}
 
+	for(int i = 0; i < BPF_MAPS_MAX; i++)
+	{
+		close(handle->m_bpf_map_fds[i]);
```

Comment:
Great catch! Can we check that each `fds > 0` too?
Also, i'd reset each entry to -1 after!

Reply:
Not all maps are populated, so I expect some of the results to be `-1`. The reset seems good to me.